scripts/webframeworks-deploy-tests/nextjs/next.config.js

```diff
@@ -1,9 +1,8 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
-  swcMinify: true,
   experimental: {
-    serverActions: true,
+    serverActions: {},
```

Comment:
The `serverActions` property has a new type: https://nextjs.org/docs/app/api-reference/next-config-js/serverActions

Reply:
I think it's not experimental anymore so we can probably remove this?